### PR TITLE
issues.js: Add support for removing labels.

### PR DESCRIPTION
**Pull request by lonerz**

### CODE REVIEW

File: src/issues/removeLabels.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    });
+    github.issues.replaceAllLabels({ // replace labels without removed labels
+      owner: repoOwner,
+      repo: repoName,
+      number: issueNumber,
+      labels: issueLabels
+    })
+    .catch(console.error)
+    .then((response) => {
+      const rejectedLabelString = rejectedLabels.join(', ');
```
Comment (synicalsyntax) [inline src/issues/removeLabels.js]: `const rejectedLabelsString = rejectedLabels.join(', ');` (note the "s" after "Label")

Comment (lonerz) [inline src/issues/removeLabels.js]: Not sure what you mean here.

Comment (synicalsyntax) [inline src/issues/removeLabels.js]: The variable `rejectedLabelString` is undefined when you try and print the comment; it should be defined as `RejectedLabelsString`

Comment (lonerz) [inline src/issues/removeLabels.js]: Oh.. typo. I pushed out a new PR.

**Pull request merged**